docs/.wordlist.txt

```diff
@@ -12,6 +12,8 @@ Juju
 Kubernetes
 lifecycle
 MicroK
+microk8s
```

Comment:
It looks like spelling checks still fail because of the number in Microk**8**s, we may need to add a separate entry for "Microk"

Reply:
I thought it was done that way on purpose to ensure we only spelled it `MicroK8s` in the docs, rather than mixing the two cases.  I corrected my new line where I called it `Microk8s`